xtest/conftest.py

```diff
@@ -1016,3 +1016,112 @@ def otdf_client_scs(otdfctl: abac.OpentdfCommandLineTool) -> abac.SubjectConditi
         ],
     )
     return sc
+
+
+@pytest.fixture(scope="module")
+def obligation_setup_no_scs_unscoped_trigger(
+    otdfctl: abac.OpentdfCommandLineTool,
+) -> tuple[abac.Attribute, abac.ObligationValue]:
+    """
+    Sets up an obligation test scenario with an attribute and obligation value.
+
+    Creates:
+    - A namespace for obligations testing
+    - An attribute with a "alpha" value
+    - An obligation definition with a required value
+    - An obligation value instance
+
+    Note: Subject mapping is intentionally omitted for testing scenarios
+    where obligations should prevent access.
+
+    Returns:
+        tuple[abac.Attribute, abac.ObligationValue]: The attribute and obligation value for testing
+    """
+    # Create a namespace for the test
+    ns = otdfctl.namespace_create("test-obligations.com")
```

Comment:
nit: the other features seem to use the temporary namespace fixture to avoid any possible conflict https://github.com/opentdf/tests/blob/00ca685c4dafeedd197a1f854240f0a2d3b7a4de/xtest/conftest.py#L388